Documentation/network/kubernetes/identity-management-mode.rst

```diff
@@ -0,0 +1,68 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+.. _IdentityManagementMode:
+
+***************************
+Identity Management Mode
+***************************
+
+.. note::
+    Identity Management by the Cilium Operator is a beta feature. Please provide feedback and file a GitHub issue
+    if you experience any problems.
+
+Cilium supports Cilium Identity (CID) management by either the Cilium Agents (default) or the
+Cilium Operator.
+
+When the Operator manages identities, identity creation is centralized. This provides benefits
+such as reduced CID duplication, which can occur when multiple Agents simultaneously create identities for
+the same set of labels. Given that there is a limitation on the maximum number of identities in a cluster
+and eBPF Policy Map size (see :ref:`bpf_map_limitations`), when the operator manages identities, we can improve the
+reliability of network policies and cluster scalability.
+
+.. note::
+
+    Labels relevant to identity management may be configured in the Cilium ConfigMap (see: :ref:`identity-relevant-labels`).
+    If the Cilium Operator is managing identities, both the Operator and Agents must be restarted to pick up the new label
+    pattern setting.
+
+Enable Identity Management by the Cilium Operator
+==================================================
+
+The Cilium Agents manage CIDs by default. This section describes the steps necessary for enabling CID management by the Cilium Operator.
+
+Enabling Operator Managing Identities on a New Cluster
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+To enable the Cilium Operator to manage identities on a new cluster, set the ``identityManagementMode`` value to ``operator`` in your Helm chart
+or set the ``identity-management-mode`` flag to ``operator`` in the ``cilium-config`` configmap.
+
+Migration Procedure
```

Comment:
nit: I would rather go with a more procedural sentence: `How to migrate to Cilium operator managing identities`